R/param_engine_party.R

```diff
@@ -0,0 +1,55 @@
+#' Tuning parameter objects for conditional tree models
```

Comment:
The documentation here is different than the documentation in all the other `param_engine_*.R` files. Should that be consistent?